packages/components/psammead-timestamp/src/index.jsx

```diff
@@ -5,14 +5,11 @@ import {
   GEL_SPACING_HLF,
   GEL_SPACING_DBL,
 } from '@bbc/gel-foundations/spacings';
-import {
-  GEL_BREVIER,
-  GEL_FF_REITH_SANS,
-} from '@bbc/gel-foundations/typography';
+import { GEL_FF_REITH_SANS } from '@bbc/gel-foundations/typography';
 import { C_CLOUD_DARK } from '@bbc/psammead-styles/colours';
 
 const StyledTimestamp = styled.time`
-  ${GEL_BREVIER};
+  ${props => props.typographyStyle}
```

Comment:
I wonder if this will fail if we don't pass the `typographyStyle` prop?
Shouldn't be like this `${props => (props. typographyStyle ? props.typographyStyle : '')};` ?

I am just asking because in the others components we are checking if we pass the  `script` prop before calling the GEL function (e.g.  ` ${props => (props.script ? getLongPrimer(props.script) : '')};`).